vector-clock/vclock.c

```diff
@@ -0,0 +1,84 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "../master/slavelist.h"
+#include "vclock.h"
+
+/**
+ * Create a new vector clock.
+ *
+ * All times are initialized to `0`.
```

Comment:
All *clocks* are initialized to the `0` vector

Reply:
Done.